_posts/2013-02-28-release-notes-v2.10.1-RC2.md

```diff
@@ -342,7 +342,7 @@ Issue(s) | Commit | Message
 [SI-6288](https://issues.scala-lang.org/browse/SI-6288) | [286dced](https://github.com/scala/scala/commit/286dced) | <notextile>SI-6288 Remedy ill-positioned extractor binding.</notextile>
 [SI-6288](https://issues.scala-lang.org/browse/SI-6288) | [f69b846](https://github.com/scala/scala/commit/f69b846) | <notextile>SI-6288 Fix positioning of label jumps</notextile>
 [SI-6288](https://issues.scala-lang.org/browse/SI-6288) | [79a43d7](https://github.com/scala/scala/commit/79a43d7) | <notextile>SI-6288 Position argument of unapply</notextile>
-[SI-6758](https://issues.scala-lang.org/browse/SI-6758) | [089173d](https://github.com/scala/scala/commit/089173d) | <notextile>Fixes SI-6758: force LazyAnnnotationInfo for DefDef and TypeDef</notextile>
+[SI-6758](https://issues.scala-lang.org/browse/SI-6758) | [089173d](https://github.com/scala/scala/commit/089173d) | <notextile>Fixes SI-6758: force LazyAnnotationInfo for DefDef and TypeDef</notextile>
```

Comment:
I understand that things can be release notes, and some projects don't like retroactive changes to such files. I'm happy to drop things as requested. I personally prefer being able to understand what I'm reading, hence offering fixes to dated content.